site/docs/guide/errors.md

```diff
@@ -100,7 +100,7 @@ If you divide your code base into different parts, _error boundaries_ allow you
 They achieve this by letting you fence errors in a part of your middleware.
 In other words, if an error is thrown in a specially protected part of middleware, it will not be able to escape from that part of the middleware system.
 Instead, a dedicated error handler is invoked, and the surrounded part of the middleware pretends to complete successfully.
-This is a feature of grammY's middleware system, so error boundaries don't care whether you're running your bot with webhooks or long polling.
+This is a feature of grammY’s middleware system, so error boundaries don’t care whether you’re running your bot with webhooks or long polling.
```

Comment:
Should this be applied consistently to all articles? I do understand if you don't think that this would be worth the effort.

Reply:
Do you want to perform these changes for all articles in a separate PR then?